product_supplierinfo_for_customer_sale_quick/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_customerinfo_sale_quick_manager,product.customerinfo.quick.sale_manager,product_supplierinfo_for_customer.model_product_customerinfo,sales_team.group_sale_manager,1,1,1,1
+access_product_customerinfo_sale_quick_user,product.customerinfo.sale.quick_user,product_supplierinfo_for_customer.model_product_customerinfo,sales_team.group_sale_salesman,1,0,0,0
```

Comment:
So, i should delete de file?

Reply:
If you don't need to add other rules, yes. You'll find a warning in the logs if there are modules without an access rule. (You'll also have issues when testing the module, because you won't be able to access those models)